honeybee_energy/hvac/doas/_base.py

```diff
@@ -58,9 +58,10 @@ class _DOASBase(_TemplateSystem):
         * demand_controlled_ventilation
         * doas_availability_schedule
         * schedules
+        * user_data
     """
     __slots__ = ('_sensible_heat_recovery', '_latent_heat_recovery',
-                 '_demand_controlled_ventilation', '_doas_availability_schedule')
+                 '_demand_controlled_ventilation', '_doas_availability_schedule', '_user_data')
```

Comment:
Remove from `__slots__` since it's already in the parent class.

Reply:
commit: doas /idealair slots edit